components/shared/components/Widget/components/Widget.tsx

```diff
@@ -74,6 +87,21 @@ export const Widget = withStaticProps(async ({ preview }: { preview: boolean })
     dispatch(fetchReferralsAction.started(undefined));
   }, [dispatch]);
 
+  const paymentMethods = [
+    {
+      id: PaymentMethod.BANK,
+      text: widget.payment_method_selector_bank_text,
+    } as const,
+    ...(vipps
```

Comment:
I _think_ I understand what's going on here, but it requires some mental gymnastics.

We're using the spread operator to untangle the inner arrays. But why?

Would it not be easier to solve this with a simple if statement and pushing to the array? In any case it's rather hard to understand what's going on here anyways, so some clarification would be nice :)

Reply:
In reference to my comment above, this would also be much easier if we got an array of payment method configurations passed down, right?